lib/index.js

```diff
@@ -18,20 +18,50 @@ var Appboy = module.exports = integration('Appboy')
   .global('appboy')
   .option('apiKey', '')
   .option('safariWebsitePushId', '')
+  .option('allowCrawlerActivity', false)
+  .option('doNotLoadFontAwesome', false)
+  .option('enableLogging', false)
   .option('automaticallyDisplayMessages', true)
+  .option('localization', 'en')
+  .option('minimumIntervalBetweenTriggerActionsInSeconds', 30)
+  .option('openInAppMessagesInNewTab', false)
+  .option('openNewsFeedCardsInNewTab', false)
+  .option('sessionTimeoutInSeconds', 30)
+  .option('requireExplicitInAppMessageDismissal', false)
   .option('enableHtmlInAppMessages', false)
   .option('trackAllPages', false)
   .option('trackNamedPages', false)
   .option('customEndpoint', '')
-  .tag('<script src="https://js.appboycdn.com/web-sdk/1.6/appboy.min.js">');
+  .option('version', 1)
+  .tag('v1', '<script src="https://js.appboycdn.com/web-sdk/1.6/appboy.min.js">')
+  .tag('v2', '<script src="https://js.appboycdn.com/web-sdk/2.1/appboy.min.js">');
 
+
+Appboy.prototype.initialize = function() {
+  var options = this.options;
+  var customEndpoint;
+  // Setup custom endpoints
+  if (options.customEndpoint) {
+    var endpoint = options.customEndpoint;
+    var regex = new RegExp('^(http|https)://', 'i');
+    customEndpoint = (regex.test(endpoint) ? endpoint : 'https://' + endpoint) + '/api/v3';
+  } else if (options.datacenter === 'eu') {
+    customEndpoint = 'https://sdk.api.appboy.eu/api/v3';
+  }
+  
+  if (Number(options.version) === 2) {
```

Comment:
Wait ... is `version` going to be a number setting? Are we going to let them input 230948 or some stuff?

I'm thinking it should be a dropdown, with values "v1" and "v2".

Reply:
Btw I realize we could just validate that version is between 1 and 2 inclusive. But better would be to just give them a dropdown, so the user doesn't have to think as much.

Reply:
Hey @ucarion - yep it is going to be a dropdown. We store our dropdown values as strings.

Reply:
oh! So the string is gonna be "1" or "2"? Ok then, nvm.